pkg/routeagent_driver/handlers/mtu/mtuhandler.go

```diff
@@ -19,30 +19,31 @@ limitations under the License.
 package mtu
 
 import (
+	"strconv"
 	"strings"
 
 	"github.com/pkg/errors"
 	submV1 "github.com/submariner-io/submariner/pkg/apis/submariner.io/v1"
 	"github.com/submariner-io/submariner/pkg/event"
-	"github.com/submariner-io/submariner/pkg/ipset"
 	"github.com/submariner-io/submariner/pkg/iptables"
+	netlinkAPI "github.com/submariner-io/submariner/pkg/netlink"
 	"github.com/submariner-io/submariner/pkg/routeagent_driver/constants"
 	"k8s.io/klog"
-	utilexec "k8s.io/utils/exec"
+)
+
+const (
+	// TCP MSS = MTU - TCP_H(20)-IP_H(20)-max_IpsecOverhed(80)
```

Comment:
```suggestion
	// TCP MSS = Default_Iface_MTU - TCP_H(20)-IP_H(20)-max_IpsecOverhed(80)
```

Just in case if you have to respin the PR for any additional comments, otherwise its fine in the current state.